src/compiler.rs

```diff
@@ -200,6 +200,8 @@ impl AmberCompiler {
             }
             let filename = dep_path.file_stem().unwrap().to_string_lossy();
             let path = format!("{dir_path}/{filename}.md");
+            Message::new_info_msg(format!("File generated at `{path}`."))
+                .show();
             let mut file = File::create(path).unwrap();
             file.write_all(document.as_bytes()).unwrap();
         }
```

Comment:
To fix the `//` issue and multiple info you could do this:

```suggestion
            let path = PathBuf::from(dir_path).join(format!("{filename}.md"));
            let mut file = File::create(path.clone()).unwrap();
            file.write_all(document.as_bytes()).unwrap();
            paths.push(String::from(path.to_string_lossy()));
        }
        let file_text = if paths.len() > 1 { "Files" } else { "File" };
        Message::new_info_msg(format!("{file_text} generated at:\n{}", paths.join("\n")))
                .show();
```

Also since I can't collaborate on this branch (because it's your fork of Amber) I can't suggest to add the variable just before the loop (of `ast_forest`):

```rs
let mut paths = vec![];
```

Reply:
You should be able as it is open for the maintainers (all my PR are on my fork usually)